docs/guides/expectations/creating_custom_expectations/overview.md

```diff
@@ -1,18 +1,76 @@
 ---
-title: Creating custom Expectations
+title: Overview
 ---
-import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Prerequisites from './components/prerequisites.jsx'
 
+You can extend the functionality of Great Expectations by creating your own custom Expectations. You can also enrich Great Expectations as a shared standard for data quality by contributing new Expectations to the open source project.
 
-(Text goes here)
+These processes compliment each other and their steps are streamlined so that one flows into the other. Once you have created a custom Expectation, it is simple to contribute it to the open source project. This section will teach you how to do both.
 
-<Prerequisites>
+<Prerequisites></Prerequisites>
 
-- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
-OR
-- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
-    
-</Prerequisites>
+## Steps to create a custom Expectation
 
+A fully developed Expectation needs to do a lot of things:
+* Execute consistently across many types of data infrastructure
+* Render itself and its Validation Results into several different formats
+* Support Profiling against new data
+* Be maintainable, with good tests, documentation, linting, type hints, etc.
+
+In order to make development of Expectations as easy as possible, we've broken up the steps to create custom Expectations into a series of bite-sized steps. Each step can be completed in minutes. They can be completed (and contributed) incrementally, unlocking value at each step along the way.
+
+Grouped together, they constitute a Definition of Done for Expectations at each [Level of Maturity](/docs/contributing/contributing_maturity).
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+<i class="fas fa-circle" style={{color: "#dc3545"}}></i> An experimental Expectation...
+
+* Has a `library_metadata` object
+* Has a docstring, including a one-line short description
+* Has at least one positive and negative example case
+* Has core logic and passes tests on at least one `Execution Engine`
+
+<i class="fas fa-circle" style={{color: "#ffc107"}}></i> A beta Expectation...
+
+* Has basic input validation and type checking
+* Has all four statement `Renderers`: question, descriptive, prescriptive, diagnostic
+* Has default `Parameter Builders` and Domain hooks to support Profiling   
+* Has core logic that passes tests for all applicable `Execution Engines` and SQL dialects
+
+<i class="fas fa-check-circle" style={{color: "#28a745"}}></i> A production Expectation...
+
+* Passes all linting checks
```

Comment:
Does it make sense to have a bullet for "Has thorough input validation and type checking, as determined by project code standards" ? It's a little vague, but I like the idea of encouraging more of this prior to elevating to production.

Reply:
We have three other things that are similar in spirit:

* Has basic input validation and type checking (at the beta level)
* Passes all linting checks (production)
* Has passed a manual review by a code owner for code standards and style guides (production)

I figured these would cover the same substantive material. If you want to emphasize input validation and type checking, I suppose we could add a bullet, but it seems redundant to me.